client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable react/jsx-sort-props */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import '../components/layouts/project-links.css';
+import { maybeUrlRE } from '../utils';
+import { Spacer } from '../components/helpers';
+import { projectMap, legacyProjectMap } from '../resources/certAndProjectMap';
+import { Link } from 'gatsby';
```

Comment:
```suggestion
import { Spacer, Link } from '../components/helpers';
import { projectMap, legacyProjectMap } from '../resources/certAndProjectMap';
```
The helper can handle both internal an external links.